jenkinsfile-runner-steward-image/scripts/run.sh

```diff
@@ -162,6 +162,29 @@ function with_termination_log() {
   return "$rc"
 }
 
+function with_retries() {
+  local retry_interval=$1 max_retries=$2
+  local cmd=("${@:3}")
+
+  local retry=1
```

Comment:
Should be zero initially:
```suggestion
  local retry=0
```